Tests/SonarQube.Bootstrapper.Tests/BootstrapperExeTests.cs

```diff
@@ -161,6 +162,44 @@ public void Exe_PreProc_VersionCheckSucceeds_PreProcFails()
         }
 
         [TestMethod]
+        public void Exe_PreProc_InstallTargets()
```

Comment:
I mean the bootstrapper when running the post-processing phase (we've no idea what the pre-/post-processor exes will do).